server/planning/content_api/events/event.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from typing import Any, Dict, List, Optional
+from pydantic import Field, validator, root_validator
+from superdesk.core.resources import ModelWithVersions
+from planning.types import EventResourceModel
+from planning.output_formatters.utils import expand_contact_info, get_matching_products
+from planning.output_formatters.json_event import JsonEventFormatter
+from typing import Any
+
+
+class ContentAPIEventResourceModel(EventResourceModel, ModelWithVersions):
+    event_contact_info: List[Dict[str, Any]] = Field(default_factory=list)
+    products: List[Dict[str, str]] = Field(default_factory=list)
+    files: List[Dict[str, Any]] = Field(default_factory=list)
+
+    # Validators to expand fields
+    @validator("event_contact_info", pre=True)
+    def expand_contacts(cls, v):
```

Comment:
Once the above is done, we don't want these validator/expansion code here, it will belong somewhere else